sycl/test-e2e/UserDefinedReductions/user_defined_reductions_wg_size_larger_than_data_size.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %{build} -o %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
And here too.

Reply:
Will address separately to avoid the need to run another round of pre-commit CI.